packages/ui/src/components/Command/DocsSearch.tsx

```diff
@@ -49,6 +49,16 @@ export interface PageResult {
   sections: PageSection[]
 }
 
+const getDocsUrl = () => {
+  if (!process.env.NEXT_PUBLIC_SITE_URL || !process.env.NEXT_PUBLIC_LOCAL_SUPABASE) {
+    return 'https://supabase.com/docs'
+  }
+
+  const isLocal =
+    process.env.NEXT_PUBLIC_SITE_URL.includes('localhost') || process.env.NEXT_PUBLIC_LOCAL_SUPABASE
+  return isLocal ? 'http://localhost:3001/docs' : 'https://supabase.com/docs'
+}
+
```

Comment:
Why have the if statement if we are doing the same thing again with the `isLocal`

Reply:
first checks if the vars exist — if they don't, just uses supabase.com/docs. Possible someone might not have NEXT_PUBLIC_LOCAL_SUPABASE set in the docs env file